src/configuring-guacamole.md

```diff
@@ -2348,6 +2370,20 @@ help, the session will simply not be recorded.
   the `typescript-path` is not specified, recording of typescripts will be
   disabled, and this parameter will be ignored.
 
+`typescript-write-existing`
+: If this parameter is set to "true", instead of attempting to generate
+  unique file names by appending incremental numbers to the typescript name,
+  Guacamole will be allowed to write to an existing typescript file,
+  potentially ovewriting any data in that file. While this is not
+  desirable in most cases, it may be necessary in situations where the
+  typescript recording is being sent to a named pipe, FIFO buffer, or
+  other special device that is processing the data and sending it
+  elsewhere.
+
+  This parameter only has an effect if typescript recording is enable. If
```

Comment:
Same thing here - "enable" should be "enabled".

Reply:
Fixed be rebase.